### ISSUE TITLE
DL3015 triggered when `-o APT::Install-Recommends=false` given

### ISSUE THREAD

**Issue by micheloosterhof** [hadolint/hadolint#299]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

DL3015 is not triggered when running apt-get -o APT::Install-Recommends=false

### Actual behavior

DL3015 is triggered when running apt-get -o APT::Install-Recommends=false

### Steps to reproduce the behavior

Run `apt-get install -o APT::Install-Recommends=false`

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Dockerfile:61 DL3015 Avoid additional packages by specifying `--no-install-recommends`
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
Thanks, we will take care of this special case in the next version. In the meantime you can use the recommendation in the rule to avoid the warning being triggered
